### DISCUSSION TITLE
Is it possible to have an addon that brings back the “mode” tab from April fools 22?

### DISCUSSION THREAD

**Discussion by penguinmoose** [ScratchAddons/ScratchAddons#5123] [Ideas] [answered]
Not sure if this is possible so that’s why I’m not doing an issue.
 
I want to know if it is possible to bring back the “mode” tab in the header from April fools 22. This includes old times mode, 60s (or something like that) mode, and prehistoric mode too.

If so, this can be added to the cat blocks addon.

**Comment by mxmou** [comment, marked as answer]
I think this should be a separate addon, not a part of cat blocks. I already have an incomplete implementation.

**Reply by penguinmoose** [reply]
Ok. Should I make an issue for this?

**Reply by BroJac5246** [reply]
> Ok. Should I make an issue for this?

Yes.

**Reply by penguinmoose** [reply]
Ok, done. #5129
